_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,160 @@
+- the [computed][] value is at least `0.12` times its [computed][] [font-size][].
+
+## Assumptions
+
+If there is a mechanism available on the page by which letter spacing can be adjusted, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
+
+## Accessibility Support
+
+While some assistive technologies are able to set [user origin][] styles, others, such as browser extensions, are only able to set style with the [author origin][]. Such assistive technologies can not create styles with a high enough [priority][] to override a `style` attribute with the [important flag][]. If accessibility support does not include assistive technologies that override letter spacing through [author origin][], this rule should not be used.
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `div` element has a `letter-spacing` of `0.12em` which is equal to the recommended minimum, given the specified font size is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+
+	<body>
+		<div style="letter-spacing: 0.12em; !important">
+			The toy brought back fond memories of being lost in the rain forest.
+		</div>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `div` element has a `letter-spacing` of `1.5px !important` (equals `0.09375em`) which is below to the recommended minimum, given the specified font size is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+
+	<body>
+		<div style="letter-spacing: 1.5px !important;">
+			The toy brought back fond memories of being lost in the rain forest.
+		</div>
+	</body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `p` element is not [visible][] because of `display: none`.
+
+```html
+<p style="display: none">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Inapplicable Example 2
+
+This text is not [visible][] because it is positioned off screen.
+
+```html
+<p style="position: absolute; top: -999em">
+	The toy brought back fond memories of being lost in the rain forest.
+</p>
+```
+
+#### Inapplicable Example 3
+
+This `body` element does not have a `style` attribute specified.
+
+```html
+<html>
+	<style>
+		body {
+			letter-spacing: 50px;
+		}
+	</style>
+
+	<body>
+		The toy brought back fond memories of being lost in the rain forest.
+	</body>
+</html>
+```
+
+#### Inapplicable Example 4
+
+This `p` element has a [letter-spacing][] of `24px` specified via the style attribute, but does not have the [important flag][].
```

Comment:
The lack of `!important` is not relevant for the Applicability anymore. This should become a Passed Example instead.

Reply:
I agree. I was going to ask if an example with too small letter-spacing and without the !important flag should be included in the passed examples.

Reply:
Also, I'm not a fan of using fixed units in the examples, as that's not really good practice in regard accessibility. A `body { font-size: 1rem }` defaults as 16px in most browsers, and is possibly a better alternative for the examples.